### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RichardBenes**
With updatereadme branch..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # README of exp
+
+Updated on updatereadme branch.
```
Comment (rbtemp) [inline README.md:3]: Yet another line, please

Comment (RichardBenes) [reply README.md:3]: OK.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # README of exp
 
 Updated on updatereadme branch.
+Another line here.
```
Comment (rbtemp) [inline README.md:5]: They are both on the same line.

**Comment by rbtemp** [comment]
Looks good.

**Pull request merged**